e2e/pnpm_lockfiles/MODULE.bazel

```diff
@@ -88,7 +88,7 @@ npm = use_extension(
         # Dep with patch
         "lock-%s__meaning-of-life__1.0.0_%s" % (
             version,
-            "1541309197" if version == "v101" else "o3deharooos255qt5xdujc3cuq",
```

Comment:
Now that we don't strip the `patch_hash=` the v9 lockfile name is long enough it gets hashed now.